cfme/tests/openstack/infrastructure/test_roles.py

```diff
@@ -67,6 +65,7 @@ def test_roles_summary(provider, soft_assert):
             soft_assert(res, err_ptrn.format(v))
 
 
+@pytest.mark.trylast
```

Comment:
**REQUIRED** Why marking as ```trylast``` all the tests should be able to be run in any order because they can then be sent to any slave during parallelizaion. We don't use test dependence at all in the framework.